importers/data_sources/importer.py

```diff
@@ -0,0 +1,209 @@
+"""
+Loads data sources into RE.
+
+This is a simple importer, as the data_sources load data is quite simple,
+and also very similar to its ultimate form, requiring little transformation.
+
```

Comment:
can you write some brief usage docs for this module?

Reply:
Hmm, there is a README.md, is that not sufficient? (I am tweaking it a bit too.)